### ISSUE TITLE
Markdown links always expanded after vim reinstall

### ISSUE THREAD

**Issue by Smingvin** [vimwiki/vimwiki#1052]
Hello,

Is there any apparent reason that would cause markdown links to always be expanded? (showing both the anchor text and the path instead of just the anchor text)

![image](https://user-images.githubusercontent.com/7772599/99160280-d520e080-26e5-11eb-9221-c5d4265cf50e.png)

This happened to me after I reinstalled vim (something broke when I updated my Mac OS to Catalina)

Everything else functions normally, folding headers, working links, tags. Just the links don't show up correctly. Vimwiki part of my vimrc follows:

```
let g:vimwiki_list = [
			\{'path': '~/Dropbox/vimwiki/', 'template_path': '~/vimwiki/templates/','template_default': 'default', 'syntax': 'markdown', 'ext': '.md', 'custom_wiki2html': 'vimwiki_markdown', 'zettel_template': "~/mytemplate.tpl", 'template_ext': '.tpl', 'links_space_char': '_'},
			\{'path': '/Users/me/Dropbox/vimwiki/radgost', 'syntax': 'markdown', 'ext': '.md', 'radgost_wiki3html': '~/wiki2html.sh', 'links_space_char': '_'},
			\{'path': '~/Dropbox/pustuwiki/', 'syntax': 'markdown', 'ext': '.md', 'corpustu_wikihtml': '~/wiki2html.sh'}
			\]
let g:zettel_options = [{},{"front_matter" : {"tags" : ""}, "template" :  "~/mytemplate.tpl"}]
```

Vimwiki version gets reported as "Unknown"

**Comment by tinmarino** [comment]
Is Vimwiki loaded ?
If ft vimwiki with syntax makrdown ?
Is foldmethid >= 2?

**Comment by Smingvin** [comment]
It must be loaded since everything else seems to work (following links, creating todo items, tagging, searching)

I don't understand the second question. The syntax is markdown, if that's the question.

How do I check which foldmethod is currently applied?

Thanks.

**Comment by Smingvin** [comment]
By the way, the version of vim I am running is 8.1.2292
Also, I just reinstalled vimwiki to stable version 2.5 (the issue persists)

**Comment by tinmarino** [comment]
Please try the dev branch and `set conceallevel` (this is what I wanted
when I said fold errounously)

**Comment by tinmarino** [comment]
An try to remove zettel, and get a minimal conf to avoid interferences

**Comment by Smingvin** [comment]
Okay, solved. 

The default version of vim that comes with mac OS Catalina is compiled without the 'conceal' feature necessary to show links properly. This can be checked by launching vim and typing `:version`. Mine reported `-conceal` when it should have said `+conceal`

Even though I said I reinstalled vim from homebrew, apparently I missed a step, you are supposed to link with:

`brew link vim`

Upon first trying this I got some error  that some file isn't writable so I had to run this:

`sudo chown -R $(whoami) /usr/local/share`

Then the linking worked, and links inside vimwiki continued to work as normal.

Interestingly, I have access to an older iMac that's running Yosemite. I checked the default version of vim and it's also without conceal.

As far as I remember, before I updated to Catalina, I ran the default system vim on Mojave. Links worked normally, indicating that it was compiled with the conceal feature. Then again, maybe my memory is faulty.
